live/liveview.go

```diff
@@ -839,6 +869,17 @@ func Redirect(ctx context.Context, url *url.URL) error {
 	return nil
 }
 
+// PushEvent sends an event to the View which a Hook can respond to
+func PushEvent(ctx context.Context, e Event) error {
+	s := socketValue(ctx)
+	if s == nil {
+		return nil
```

Comment:
should sending on a nil socket return an error? (Presumably a sentinel error, so it can be checked for.)

Reply:
maybe?  we don't elsewhere but that can change.  is it more idiomatic to do so?

Reply:
I guess it depends on what semantics you want. :)

If the socket is likely to be nil in a way that doesn't signify a developer error, then we should leave it as it is.

 If the socket being nil signifies that the developer has messed something up, then silently ignoring that error will result in it being harder to debug what is going wrong.

 Either way, it's something that we can change in a separate PR ( if we want to)